resources/PythonPip3Dsc/PythonPip3Dsc.psm1

```diff
@@ -104,8 +162,13 @@ function Invoke-Pip3Install {
         [Parameter()]
         [string]$Version,
 
+        # not explicitly used, only to call from lower functions if parameters are passed
```

Comment:
This comment doesn't seem accurate and can probably be removed. If $IsUpdate and $DryRun are present then it does affect the final command argument right?

Reply:
You are right, I removed it.